libraries/standard/mqtt/lexicon.txt

```diff
@@ -0,0 +1,229 @@
+ack
+acked
+acks
+addrecord
+api
+app
+aws
+bufferlength
+bytesreceived
+bytestoread
+bytestoreceive
+bytestorecv
+bytestosend
+bytestowrite
+ca
+calculatestatepublish
+cbmc
+cleansession
+cmock
+com
+config
+connack
+connectinfo
+connectpacketsize
+const
+createcleansession
+currentstate
+defragmenting
+deserialization
+deserialize
+deserializeack
+deserialized
+deserializepublish
+deserializing
+didn
+disconnectpacketsize
+doesn
+dup
+emptyindex
+endif
+enums
+expectprocessloopcalls
+fixedbuffer
+getconnectpacketsize
+getdisconnectpacketsize
+getincomingpackettypeandlength
+getpacketid
+getpingreqpacketsize
+getpublishpacketsize
+getsubscribepacketsize
+gettime
+getunsubscribepacketsize
+handleincomingack
+handleincomingpublish
+handlekeepalive
+hasn
+headersize
+https
+ifndef
+inc
+incomingpacket
+incomingpublish
+init
+int
+iot
+isn
+lwt
+malloc
+managekeepalive
+memcpy
+milli
+min
+misra
+modifyincomingpacket
+mosquitto
+mqtt
+mqttbadparameter
+mqttbadresponse
+mqttconnectinfo
+mqttcontext
+mqttfixedbuffer
+mqttillegalstate
+mqttkeepalivetimeout
+mqttnodataavailable
+mqttnomemory
+mqttpacketinfo
+mqttpubackpending
+mqttpubacksend
+mqttpubacktype
+mqttpubcomppending
+mqttpubcompsend
+mqttpublishdone
+mqttpublishinfo
+mqttpublishsend
+mqttpubrecpending
+mqttpubrecsend
+mqttpubrelpending
+mqttpubrelsend
+mqttqos
+mqttrecvfailed
+mqttsendfailed
+mqttserverrefused
+mqttstatecollision
+mqttstatenull
+mqttsubscribeinfo
+mqttsuccess
+msb
+networkbuffer
+networkcontext
+newstate
+nextpacketid
+noninfringement
+openssl
+optype
+org
+packetid
+packetidentifier
+packetsize
+packettype
+param
+paramters
+payloadlength
+pbuffer
+pbuffertosend
+pcallbacks
+pconnectinfo
+pcontext
+pcurrentstate
+pcursor
+pdestination
+pem
+pfixedbuffer
+pheadersize
+pincomingpacket
+pingreq
+pingresp
+pingresps
+pingresptimeoutms
+pmessage
+pmqttcontext
+pnetworkbuffer
+pnetworkcontext
+pnewstate
+posix
+ppacketid
+ppacketinfo
+ppacketsize
+ppubinfo
+ppublishinfo
+pqos
+pre
+premainingdata
+premaininglength
+processloop
+psessionpresent
+psource
+pstate
+psubscribeinfo
+psubscriptionlist
+ptr
+ptransport
+ptransportinterface
+puback
+pubcomp
+publishstate
+pubrec
+pubrel
+pubrels
+pwillinfo
+qos
+readfunc
+receiveloop
+receivepacket
+recordcount
+recordindex
+recv
+recvfunc
+reestablishment
+remaininglength
+remainingtime
+remainingtimems
+resending
+responsecode
+searchstates
+sendpacket
+sendpublish
+sendpublishacks
+serializeack
+serializeconnect
+serializeconnectpacket
+serializedisconnect
+serializepayload
+serializepingreq
+serializepublish
+serializepublishheader
+serializesubscribe
+serializeunsubscribe
+shoulddelete
+shouldn
+sizeof
+sourcelength
+src
+stateafterdeserialize
+strerror
+struct
+structs
+suback
+sublicense
+subscriptioncount
+subscriptiontype
+tcp
+tcpsocket
+testcase
+timeoutms
+tls
+uint
+un
```

Comment:
Does this represent a word of the kind "_un_-<word>"?

Reply:
Yes